app/Models/User_Role.ts

```diff
@@ -0,0 +1,22 @@
+import { DateTime } from 'luxon'
+import { BaseModel, column, ManyToMany, manyToMany } from '@ioc:Adonis/Lucid/Orm'
+
+export default class User_Role extends BaseModel {
```

Comment:
why is file and class name combination of title and snake case?

just name it `UserRole`